components/content/ZoneMap.tsx

```diff
@@ -5,13 +5,27 @@ import MapView, {MAP_TYPES, MapViewProps, PoiClickEvent, Region} from 'react-nat
 import {RegionBounds, regionFromBounds, updateBoundsToContain} from 'components/helpers/geographicCoordinates';
 import {AvalancheForecastZonePolygon, toLatLngList} from 'components/map/AvalancheForecastZonePolygon';
 import {useToggle} from 'hooks/useToggle';
-import {AvalancheCenterID, DangerLevel, Geometry} from 'types/nationalAvalancheCenter';
+import {AvalancheCenterID, DangerLevel, Geometry, MapLayerFeature} from 'types/nationalAvalancheCenter';
 
 const defaultAvalancheCenterMapRegionBounds: RegionBounds = {
   topLeft: {latitude: 0, longitude: 0},
   bottomRight: {latitude: 0, longitude: 0},
 };
 
+export const MapViewZoneFor = (center: AvalancheCenterID, feature: MapLayerFeature): MapViewZone => {
```

Comment:
nit: idiomatic usage would reserve a capitalized name for a React component, or a class. 
```suggestion
export const mapViewZoneFor = (center: AvalancheCenterID, feature: MapLayerFeature): MapViewZone => {
```